src/powscript.bash

```diff
@@ -163,6 +168,11 @@ evaluate(){
   evalstr=""
 }
 
+tosh(){
+  runtime=sh
+  cat - | transpile_sh
```

Comment:
Keep in mind that the stdin is already implicitly passed through function calls, e.g.:
```shell
f() {
  cat
}

g() {
  f
}

echo x | g #>> outputs 'x'
```